docs/_releases/v4.1.4/stubs.md

```diff
@@ -261,15 +261,18 @@ The Promise library can be overwritten using the `usingPromise` method.
 Causes the stub to throw an exception (`Error`).
 
 
-#### `stub.throws("TypeError");`
+#### `stub.throws("Error msg");`
 
-Causes the stub to throw an exception of the provided type.
+Causes the stub to throw an exception with the provided message.
```

Comment:
That's incorrect. The `Error#message` is not set, but the `Error#name` is. See this example:
```javascript
> stub=sinon.stub().throws("An_error")
> stub2=sinon.stub().throws(new TypeError("The message") )
> try { stub() } catch (err) { console.log(' err.name', err.name, '\n', 'err.message', err.message);   }
 err.name An_error 
 err.message 
undefined
> try { stub2() } catch (err) { console.log(' err.name', err.name, '\n', 'err.message', err.message);   }
 err.name TypeError 
 err.message The message
```

I suggest just saying it like it is: 
> ".... an exception with the `name` property set to the provided string"

Reply:
Also, the second parameter is not exposed in the docs, so an alternative to adding it could be to just mention it for this one:

```markdown
#### `stub.throws("ErrorType", "Optional error message")

Causes the stub to throw an `Exception` with the `name` property set to the type argument. The message argument is optional and will set the `message` prop of the error.
```

Reply:
Thanks for the feedback - I expect to update this tomorrow (Monday) evening.